packages/venia-concept/src/components/CategoryList/categoryTile.css

```diff
@@ -6,22 +6,21 @@
 }
 
 .imageWrapper {
+    background-image: var(--venia-image);
+    background-position: 50% 50%;
+    background-size: cover;
     border-radius: 50%;
-    border: 1px solid rgb(var(--venia-border));
+    box-shadow: 0 0 0 1px rgb(var(--venia-border));
     display: block;
     height: 5rem;
     margin: 0 auto 1rem auto;
-    overflow: hidden;
-    padding: 0.1rem;
     width: 5rem;
 }
 
 .image {
-    border-radius: 50%;
-    height: auto;
-    max-height: 100%;
-    max-width: 100%;
-    width: auto;
+    height: 100%;
+    opacity: 0;
```

Comment:
Hide the image, but not from screen readers. It should be there, and if you right click the tile, the context menu should give you actions related to the image element (e.g., `View image`).